dvc/output.py

```diff
@@ -1017,7 +1018,8 @@ def get_dir_cache(self, **kwargs):
         except FileNotFoundError:
             if self.remote:
                 kwargs["remote"] = self.remote
-            self.repo.cloud.pull([obj.hash_info], **kwargs)
+            with suppress(Exception):
```

Comment:
looks scary, btw :( it's quite sad that there is no way in all these Python libs ecosystem to handle these things properly w/o an extreme pain

Reply:
It looks scary because the arch is shit here and we shouldn't be doing this implicit action like this. Not because python.